Dockerfile

```diff
@@ -26,83 +16,84 @@ RUN python -m venv /venv
 RUN pip install --upgrade "pip==21.3.1"
 RUN useradd -d /app -M --uid 1000 --shell /usr/sbin/nologin kitsune
 
-RUN apt-get update && \
+RUN apt-get update && apt-get install apt-transport-https && \
+    curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
```

Comment:
The old image installed node (this way) - just in a separate step. I moved it earlier to take greater advantage of caching.

The reason for not using a docker node image is - at least currently - we need a python+node environment to build our staticfiles. Once we have webpack out the door, that might change (though I'm not sure exactly where our js locale files would fit in - possibly not a requirement for our javascript tests), and we could/should probably switch to using a node image to build the webpack bundle. I think this will also make splitting up the job into multiple steps in a workflow easier.